docs/it/linkedin.md

```diff
@@ -0,0 +1,66 @@
+
+**1) Indicazioni generali**
+LinkedIn, nel mondo, conta circa 830 milioni di utenti attivi, di cui circa 17 milioni in Italia (fonte: https://www.linkedin.com/pulse/utenti-linkedin-italia-e-nel-mondo-/?originalSubdomain=it).
+
+Per il settore tech è, senza dubbio, lo strumento migliore che le persone che lavorano come recruiter/head hunters/talent acquisition usano tutti i giorni per trovare professionisti/e del settore IT. Prima di scendere nel dettaglio dei vari punti della guida, alcune considerazioni generali per iniziare al meglio:
+
+- Moltissimi recruiter non conoscono il gergo tech, i nomi delle tecnologie/framework, confondono Java con JavaScript (si, succede ancora) e così via. E questo, immagino, potrebbe non farti piacere. Ma cerca comunque di essere sempre comprensivo ed empatico verso chi ti scrive. Cerca, per quanto possibile, di rispondere sempre ai messaggi che ricevi perché dall'altra parte c'è comunque una persona che sta lavorando e che merita rispetto esattamente come lo meriti tu. Se al momento non sei in cerca di un nuovo lavoro, ti consiglio magari di prepararti un testo preimpostato da inviare a chi ti scrive, da usare banalmente come risposta. Tu perdi pochi secondi di tempo ma ti assicuro che chi c'è dall'altra parte apprezzerà!
+
+- La questione dell'opentowork: sicuramente penserai che la cornice verde #opentowork non sia un granché, vero? E pensi anche che, se la attivi, la tua azienda ti sgamerà subito e capirà che stai cercando un nuovo lavoro. Non so se lo sai, ma c'è anche la possibilità di segnalare che sei opentowork ma non al "pubblico": basta cliccare sul tasto "disponibile per" sotto la foto profilo e, dopo aver selezionato l'opzione "cambiare lavoro" nella maschera che ti si aprirà, scorri in basso e troverai un menù con il titolo "scegli chi può vedere che sei disponibile per opportunità di lavoro" e seleziona "solo recruiter": in questo modo i recruiter, utilizzando Linkedin Recruiter, vedranno una voce nel tuo profilo "disponibile a lavorare".
+- Chi cerca professionisti su Linkedin lo fa principalmente utilizzando le keyword (parole chiave), in due modi: o usando le stringhe booleane o usando appunto le parole chiave ed altri filtri con la licenza LinkedIn Recruiter (il profilo a pagamento di Linkedin che sblocca tutta una serie di funzionalità, fra cui tantissimi filtri di ricerca ottimizzata e molto altro). Da qui si intuisce che TUTTO ciò che è scritto sul tuo profilo è utile a farti trovare e, soprattutto, a rientrare in modo mirato nelle ricerche che vengono effettuate sulla piattaforma proprio tramite le parole chiave. Approfondiremo comunque questo aspetto nelle sezioni successive della guida.
```

Comment:
Cosa sono le stringhe booleane? Forse converrebbe spiegare?

Reply:
Vogliamo integrare con questo link?
https://www.linkedin.com/help/linkedin/answer/a524335

Reply:
Ah ora ho capito! Si dai ci sta per evitare magari di entrare troppo in tecnicismi